woocommerce/wc-admin
# Add default marketing extensions as fallbacks

Adds the marketing extensions for the "reach" and "grow" categories used in the task list.

### Screenshots

<img width="708" alt="Screen Shot 2021-08-05 at 9 15 37 AM" src="https://user-images.githubusercontent.com/10561050/128356115-66a8901e-cece-4fff-b08b-d8e0f9a8d6c5.png">


### Detailed test instructions:

* Delete the `woocommerce_admin_remote_free_extensions_specs` transient
* Change the data source in `src/Features/RemoteFreeExtensions/DataSourcePoller.php` to an invalid URL
* Visit the task list marketing task to confirm that extensions exist
